user_service/config/config.go

```diff
@@ -1 +1,39 @@
 package config
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+var AppConfig struct {
+	POSTGRES_USER     string
+	POSTGRES_DB_NAME  string
+	POSTGRES_PASSWORD string
+	SENDER_EMAIL      string
+	SENDER_APP_PASS   string
+	SMTP_SERVER       string
+	SMTP_PORT         int
+	SECRET_KEY        string
+}
+
+func Init() {
+	if err := godotenv.Load(); err != nil {
+		// Handle error loading .env file
+		panic(err)
```

Comment:
Okay that's fine, the idea is that the application should not panic if cannot find a .env file.
Because environment variables are not always fed through .env file. There are other ways as well.